### ISSUE TITLE
Allow CookiesFilter to reinit if needed

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#22312]
In the init method, the `CookiesFilter` sets tomcat's secure cookie flag to `true`.  If the filter reinits, this filter tries to set the  secure cookie flag again, which throws an error, and the `CookiesFilter` filter does not get added to the chain.  Now, this is not a huge deal as re-initing tomcat's filter chain is something that does not happen unless forced by java code - like the code found here: https://github.com/wezell/com.dotcms.servlet.filter.tomcat/blob/master/src/main/java/com/dotmarketing/osgi/servlet/TomcatServletFilterUtil.java#L144-L172
Still, it would be cool if we could cleanly and dynamically add filters via OSGi and this filter is the only thing that prevents a clean reinit of the FilterChain.

**Comment by freddyucv** [comment]
**Passed Internal QA**: 

I used this plugins to thesi this issue:  https://github.com/wezell/com.dotcms.servlet.filter.tomcat


- First I uncomment this line

https://github.com/wezell/com.dotcms.servlet.filter.tomcat/blob/master/src/main/java/com/dotmarketing/osgi/servlet/Activator.java#L21

https://github.com/wezell/com.dotcms.servlet.filter.tomcat/blob/master/src/main/java/com/dotmarketing/osgi/servlet/Activator.java#L32

and generated the plugins jar.

- Later I uploaded the plugins in 22.06 and I got the follow error in the log file
```

dotcms_1         | 25-Jul-2022 22:25:21.206 SEVERE [fileinstall-/data/shared/felix/load] org.apache.catalina.core.StandardContext.filterStart Exception starting filter [CookiesFilter]
dotcms_1         | 	java.lang.IllegalStateException: Property [secure] cannot be added to SessionCookieConfig for context [] as the context has been initialised
dotcms_1         | 		at org.apache.catalina.core.ApplicationSessionCookieConfig.setSecure(ApplicationSessionCookieConfig.java:145)
dotcms_1         | 		at com.dotmarketing.filters.CookiesFilter.init(CookiesFilter.java:38)
dotcms_1         | 		at org.apache.catalina.core.ApplicationFilterConfig.initFilter(ApplicationFilterConfig.java:272)
dotcms_1         | 		at org.apache.catalina.core.ApplicationFilterConfig.getFilter(ApplicationFilterConfig.java:253)
dotcms_1         | 		at org.apache.catalina.core.ApplicationFilterConfig.<init>(ApplicationFilterConfig.java:102)
dotcms_1         | 		at org.apache.catalina.core.StandardContext.filterStart(StandardContext.java:4613)
dotcms_1         | 		at com.dotmarketing.osgi.servlet.TomcatServletFilterUtil.addFilter(TomcatServletFilterUtil.java:172)
dotcms_1         | 		at com.dotmarketing.osgi.servlet.Activator.start(Activator.java:21)
dotcms_1         | 		at org.apache.felix.framework.util.SecureAction.startActivator(SecureAction.java:697)
dotcms_1         | 		at org.apache.felix.framework.Felix.activateBundle(Felix.java:2240)
dotcms_1         | 		at org.apache.felix.framework.Felix.startBundle(Felix.java:2146)
dotcms_1         | 		at org.apache.felix.framework.BundleImpl.start(BundleImpl.java:998)
dotcms_1         | 		at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundle(DirectoryWatcher.java:1260)
dotcms_1         | 		at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundles(DirectoryWatcher.java:1233)
dotcms_1         | 		at org.apache.felix.fileinstall.internal.DirectoryWatcher.doProcess(DirectoryWatcher.java:520)
dotcms_1         | 		at org.apache.felix.fileinstall.internal.DirectoryWatcher.process(DirectoryWatcher.java:365)
dotcms_1         | 		at org.apache.felix.fileinstall.internal.DirectoryWatcher.run(DirectoryWatcher.java:316)
```

- Finally I test in 22.07 and I did not get the error when I uploaded the plugins

**Comment by bryanboza** [comment]
Fixed, tested on release-22.07 // Docker // FF. 

Now we are able to upload the plugin without problems.
